src/helpers/defaultRules.js

```diff
@@ -100,7 +100,7 @@ exports.defaultRulesFilter = async function defaultRulesFilter({
 }) {
 	try {
 		const [userDetails, defaultRules] = await Promise.all([
-			getUserDetailsFromView(requesterId, isAMentor(roles)),
+			getUserDetails(requesterId, isAMentor(roles)),
```

Comment:
@sumanvpacewisdom one place only ?

Reply:
@rakeshSgr Among two defaultRules functions only one is using the view , rest all are using user table directly